scripts/cmakegen/cmakegen.py

```diff
@@ -39,12 +39,13 @@ def __init__(self, features):
     def generate(self, language):
         if self._features is None:
             return str()
-        template_c = 'set(WITH_EXTENSION_%extension OFF)\n'
-        template_cpp = 'set(KAA_WITHOUT_%extension 1)\n'
+        template_c = '\nset(WITH_EXTENSION_%extension OFF)'
+        template_cpp = '\nset(KAA_WITHOUT_%extension 1)'
+        comment = '# Disable unused features' if len(self._features) < 4 else str()
```

Comment:
Why?

Also, the condition seems to be flipped. Are you emitting comment when there are 0-3 disabled features? (I would do if there are 1+ of them.)